changelog/6296.improvement.md

```diff
@@ -0,0 +1,12 @@
+* Modified functionality of ``SingleStateFeaturizer`` 
+
+``SingleStateFeaturizer`` uses trained NLU ``Interpreter`` to featurize intents and action name. This modified ``SingleStateFeaturizer`` can replicate ``LabelTokenizerSingleStateFeaturizer`` functionality. This component is deprecated from now on. 
+To replicate ``LabelTokenizerSingleStateFeaturizer`` functionality, add a ``Tokenizer``  with ``intent_tokenization_flag: True`` and ``CountVectorsFeaturizer`` to the NLU ``pipeline``. Please update your configuration file. 
+
+The new ``SingleStateFeaturizer`` stores slots, entities and forms in sparse features for more lightweight storage. 
+
+``BinarySingleStateFeaturizer`` will be deprecated in the near future, so, we recommend to switch to ``SingleStateFeaturizer`` when possible. 

```

Comment:
I think we already deprecated `BinarySingleStateFeaturizer`.

Reply:
it's not fully gone yet -- if you specify it in your config it will work.

Reply:
Yes, but that is exactly what deprecated is for :). It still works but at some point it will be removed in the future. So I would change the sentence to 
```BinarySingleStateFeaturizer`` is deprecated and will be removed in the future. We recommend to switch to ``SingleStateFeaturizer``.`